src/containers/Ledger/index.js

```diff
@@ -248,7 +348,7 @@ class Ledger extends Component {
     const loader = loading ? <Loader className="show" /> : <Loader />
 
     return (
-      <div className="ledger-page">
```

Comment:
Please keep this `ledger-page`.  `ledger` is too broad for an entire page class.